gaeguli/meson.build

```
@@ -27,6 +27,8 @@ install_headers(source_h, subdir: gaeguli_install_header_subdir)
 gaeguli_c_args = [
   '-DG_LOG_DOMAIN="GAEGULI"',
   '-DGAEGULI_COMPILATION',
+  '-DGAEGULI_PIPELINE_WORKER_ALT="' + join_paths(meson.current_build_dir(), 'workers', 'pipeline_worker','gaeguli_pipeline_worker') +'"',
+  '-DGAEGULI_TARGET_WORKER_ALT="' + join_paths(meson.current_build_dir(), 'workers', 'target_worker','gaeguli_target_worker') + '"'
```

You forgot to update this too.

```suggestion
  '-DGAEGULI_PIPELINE_WORKER_ALT="' + join_paths(meson.current_build_dir(), 'workers', 'gaeguli_pipeline_worker') +'"',
  '-DGAEGULI_TARGET_WORKER_ALT="' + join_paths(meson.current_build_dir(), 'workers', 'gaeguli_target_worker') + '"'
```